### PR TITLE
[DO NOT MERGE]Add mk22_stacklight_basic model

### PR DESCRIPTION
**Pull request by dkalashnik**

### CODE REVIEW

File: classes/cluster/mk22_stacklight_basic/fuel/config.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- service.git.client
+- system.linux.system.single
+- system.linux.system.repo.tcp_salt
+- system.openssh.client.lab
+- system.salt.master.git
+- system.reclass.storage.salt
+- system.salt.minion.pki.authority
+- system.salt.minion.pki.certificate
+- system.sphinx.server.doc.reclass
+- system.keystone.client.single
```
Comment (cznewt) [inline classes/cluster/mk22_stacklight_basic/fuel/config.yml]: move these to ctl01 please, that will be standard way to deliver client roles [1st member of each cluster]
```
  reclass:
    storage:
      node:
        openstack_control_node01:
          classes:
          - client.class
```

Comment (dkalashnik) [inline classes/cluster/mk22_stacklight_basic/fuel/config.yml]: Done, moved to openstack_control_node01:
```
- system.keystone.client.single
- system.keystone.client.service.nova
- system.mysql.client.single
```

**Comment by dkalashnik** [comment]
@cznewt fixed

**Comment by elemoine** [comment]
@dkalashnik I think you can close this.

**Comment by dkalashnik** [comment]
@elemoine Yep, it has sense

**Pull request closed without merging**